### ISSUE TITLE
Import Competitor Formats

### ISSUE THREAD

**Issue by mmcguill** [mmcguill/Strongbox#125]
- 1Password
- LastPass
- Dashlane
- Encryptr
- Improved CSV
- Others?

**Comment by mlfreeman2** [comment]
Chrome & Firefox password stores (or export format).

Also, I don’t remember if it was on the LastPass or 1Password forums, but someone had an AppleScript that drove the Safari passwords UI to dump my iCloud Keychain passwords out to a file.

I doubt Apple would ever approve such functionality in a iStore app, so you’d have to probably keep such a tool only here on Github (even if you just mirrored the one out there already).

Ultimately that might just produce an “improved CSV” file anyway.

**Comment by mmcguill** [comment]
Yeah, I guess these kind of homemade scripts could be useful for anyone importing, and will probably always be necessary given the sheer number of formats out there. Could be worth putting together/collating/curating a list of conversion tools/scripts, or maybe there's one out there already?

**Comment by mlfreeman2** [comment]
If I can find the source for the script I used, I’ll post a link on this ticket.

**Comment by georgesnow** [comment]
Mr c or something it’s on 1p forums

**Comment by georgesnow** [comment]
I used it to convert from old password manager to keepass

**Comment by georgesnow** [comment]
https://discussions.agilebits.com/discussion/101693/moving-to-1password-from-another-password-manager

**Comment by marcelstoer** [comment]
This is somewhat related to #102
